pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -206,3 +206,59 @@ Examples:
 	cmd.Long = fmt.Sprintf(longDesc, fullName)
 	return cmd
 }
+
+// NewCmdStop is a wrapper for the Kubernetes cli stop command
+func NewCmdStop(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	cmd := cmd.NewCmdStop(f.Factory, out)
+	longDesc := `Gracefully shut down a resource by id or filename.
+
+Attempts to shut down and delete a resource that supports graceful termination.
+If the resource is resizable it will be resized to 0 before deletion.
+
+Examples:
+
+	# Shut down foo.
+	$ %[1]s stop replicationcontroller foo
+
+	# Stop pods and services with label name=myLabel.
+	$ %[1]s stop pods,services -l name=myLabel
+
+	# Shut down the service defined in service.json
+	$ %[1]s stop -f service.json
+
+	# Shut down all resources in the path/to/resources directory
+	$ %[1]s stop -f path/to/resources
+`
+	cmd.Long = fmt.Sprintf(longDesc, fullName)
```

Comment:
@smarterclayton fyi after the [latest](https://github.com/GoogleCloudPlatform/kubernetes/commit/e1256c08027748c4a0dc8078f1f8db5afb673923) change in stop code we can use `osc stop` as is since if the passed resource doesn't implement the Reaper, stop will fallback to delete (https://github.com/GoogleCloudPlatform/kubernetes/blob/e1256c08027748c4a0dc8078f1f8db5afb673923/pkg/kubectl/cmd/delete.go#L112). 

Regarding the implementation, DC first has to support the Resizer and then the Reaper. How are we going to implement those since DC is OpenShift-specific resource and we need to make that change in Kubernetes?
https://github.com/GoogleCloudPlatform/kubernetes/blob/e1256c08027748c4a0dc8078f1f8db5afb673923/pkg/kubectl/stop.go#L51
https://github.com/GoogleCloudPlatform/kubernetes/blob/e1256c08027748c4a0dc8078f1f8db5afb673923/pkg/kubectl/resize.go#L92